homeassistant/components/radarr/sensor.py

```diff
@@ -1,21 +1,22 @@
 """Support for Radarr."""
+from datetime import datetime, timedelta
 import logging
 import time
-from datetime import datetime, timedelta
 
+from pytz import timezone
```

Comment:
This doesn't seem needed. It's currently used to convert a `datetime.tzinfo` instance to a string and then back to a `datetime.tzinfo` instance.

I think we can remove that line and this import.